README.md

```diff
@@ -33,6 +33,18 @@ If you don't want to generate the report file, but are interested in just runnin
 make tests
 ```
 
+## Adding a new tool
+* Add the tool to the [SymbiFlow/conda-packages](https://github.com/SymbiFlow/conda-packages) repository to make it avaliable as a `conda` package.
+* Add the package as a dependency in `conf/environment.yml`.
+* Add the tool as a submodule to this repository via `git submodule add <git_url> third_party/tools/<name>`.
```

Comment:
I don't think we need this step if the package is in conda?

Reply:
right, there is no hard requirement that all tools must be added to `conda-packages`
I've modified this point accordingly